system_files/usr/share/ublue-os/privileged-setup.hooks.d/20-dx.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+source /usr/lib/ublue/setup-services/libsetup.sh
+
+version-script dx-usergroups-lts privileged 1 || exit 0
```

Comment:
```suggestion
version-script dx-usergroups privileged 1 || exit 0
```
I don't think we need "lts" here

Reply:
@tulilirockz What do you think. This is pretty much a blackbox for me, so your call

Reply:
Oh LOL yeah, that "dx-usergroups" is just an arbitrary label,  check out how the `version script` thing works! (has a teensy tiny bit of docs there)